Core/Services/CarService.cs

```diff
@@ -73,14 +73,21 @@ private async Task<bool> IsCarExist(Car newCar)
                     newCar.TechnicalPassport));
         }
 
-        public async Task<bool> CheckIsCarBelongsToUserByIds(int carId, string userId)
+        public async Task<bool> CheckIsUserVerifiedByIdsAsync(int carId, string userId)
         {
-            return await _carRepository.AnyAsync(new CarSpecification.GetByIds(carId, userId));
+            var car = await _carRepository.GetBySpecAsync(new CarSpecification.GetByIds(carId, userId));
+
+            ExceptionMethods.CarNullCheck(car);
+
+            return car.IsVerified;
         }
 
-        public async Task<bool> CheckIsCarVerifiedById(int carId)
+        public List<CarDTO> GetUserVerifiedByUserId(string userId)
```

Comment:
Something wrong with this name, probably we should use "GetUserVerifiedCarsByUserId"?

Reply:
I try to avoid the word "Car" because it is in the name of the service